packages/web/stories/forms/select/Select.stories.ts

```diff
@@ -83,6 +84,21 @@ Invalid.args = {
   optionValue: 'code'
 };
 
+export const Tags = Template.bind({});
+Tags.args = {
+  label: 'Label',
+  useItemTemplate: true,
+  placeholder: 'Placeholder',
+  invalid: true,
+  options: [
+    { label: 'New York', code: 'NY', tags: [ { label: 'Conteúdos'}, {label: 'Autoavaliações'},{label: 'Sessões de psicoterapia (16)'}, {label: 'Sessões de psiquiatria'}] },
+    { label: 'Rome', code: 'RM', disabled: true },
+    { label: 'London', code: 'LDN' },
+    { label: 'Istanbul', code: 'IST' },
+  ],
```

Comment:
Indentação inconsistente aqui. Sugestão:

```suggestion
  options: [
    {
      label: 'New York',
      code: 'NY',
      tags: [
        { label: 'Conteúdos'},
        { label: 'Autoavaliações' },
        { label: 'Sessões de psicoterapia (16)' },
        { label: 'Sessões de psiquiatria'},
      ],
    },
    { label: 'Rome', code: 'RM', disabled: true },
    { label: 'London', code: 'LDN' },
    { label: 'Istanbul', code: 'IST' },
  ],
```